requirements.txt

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+click>=8.1.7,<9.0.0
+httpx>=0.25.0,<1.0.0
+jinja2
+openai>=1.13.3,<2.0.0
+rouge_score
+tqdm>=4.66.2,<5.0.0
```

Comment:
can you add `instructlab` here?  until we resolve #6

Reply:
Yup, added that and a note to remove when the issue is resolved